tutorials/WORKFLOW.md

```diff
@@ -0,0 +1,35 @@
+# hydrus Workflow
+
+ This document can be used to understand how hydrus works. <br>
+To understand the concepts related to hydrus:<br>
+https://www.hydraecosystem.org/Starting-Material
+
+### How is an ```ApiDoc``` defined:
+- There are two ways of defining an ApiDoc:
+  1. Writing a Hydra-compliant JSON-LD like [here]( https://github.com/HTTP-APIs/hydrus/blob/master/hydrus/samples/doc_writer_sample_output.py)
+  1. Defining the ApiDoc programmatically using the code like in [here](https://github.com/HTTP-APIs/hydrus/blob/master/hydrus/samples/doc_writer_sample.py)
+- Any valid ApiDoc can be passed to the server so that it can parse the doc, generate a database and run the service.
+- The document containing the context is generated by the doc_maker file in the hydra_python_core repo. hydrus provides the link in its response for the context.    
```

Comment:
What exactly do you want to convey through this point?

Reply:
So these were some points that were told to me by Lorenzo as I was a little confused about how the response was generated and also Chris had told me the point at line 12, so I just thought I will add these too.